### PR TITLE
Make it work properly on heroku

### PR DESCRIPTION
**Pull request by martyzz1**
adds a runtime python 2

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -35,9 +39,11 @@ def make_pdf_from_url():
     pdf_file = make_pdf(source)
     return send_file(pdf_file, as_attachment=False)
 
+cross_origin(["www.pawsquad.com", "martinwork.eu.ngrok.io"])
```
Comment (seanbehan) [inline app.py]: Can you remove these hard coded domains and put into env vars?

**Comment by martyzz1** [comment]
Ooops! Sorry yes will do..

File: app.py (hunk 1 above)
Comment (seanbehan) [reply app.py]: Maybe something like this... 
```
# heroku config:set CORS_DOMAINS="www.pawsquad.com,martinwork.eu.ngrok.ido"
from os import environ as env
cross_origin(env.get('CORS_DOMAINS', '').split(','))
```

Mind putting something in the README about how to use the cors code you've added?

Thanks

**Comment by martyzz1** [comment]
done. untested... as I'm now using a different solution for my pdf needs,
so I don't have the testing setup for it anymore... But the cors code
worked, just need to check the env a dn split does what it says on the tin

**Comment by seanbehan** [comment]
@martyzz1 thanks for the PR! I'm curious what you were using this for/what you decided to use instead?

**Comment by martyzz1** [comment]
I just decided it made more sense to use django_pdfkit which wraps around wkhtmltopdf

**Pull request merged**